types/react-widgets/lib/DropdownList.d.ts

```diff
@@ -142,6 +142,11 @@ interface DropdownListProps extends ReactWidgetsCommonDropdownProps<DropdownList
      * @default false
      */
     busy?: boolean;
+    /**
+     * An optional ReactNode to override the spinner gif element when the busy property
+     * is set to true.
+     */
+    busySpinner?: React.ReactNode;
```

Comment:
Hey! Thanks for your job.
Do we need to bump `react-widgets` version? (https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-widgets/index.d.ts#L1)
Is this option included in version 4.3?

Reply:
@r3nya Regarding bumping version, I'm not sure.  I see v4.4.11 latest (https://www.npmjs.com/package/react-widgets), v4.2.6 latest on Git tags (https://github.com/jquense/react-widgets), and I'm not sure what other features / changes there were between 4.3 (most recent typing) and 4.4.11 or 4.2.6.

Regarding "Is this option included in version 4.3", I just found that it is _not_.  That change seems to first appear in https://github.com/jquense/react-widgets/commit/ff2c4e96791d8c8e87e6a5afc7873da377ac08f2, which correlates to 4.4.x (https://github.com/jquense/react-widgets/releases?after=react-widgets-simple-number%404.1.15).  I checked v4.4.0 and v4.4.2 and it _did_ have busySpinner.

Summary: as of v4.4.0, busySpinner appeared; v4.4.2 was created the same day, so that would be my recommendation for version to attach this change to.

I am not sure what other typing changes are missing from the v4.4.2, but this should improve typing for v4.4.2 over what is there now.

Does that help? :)

Reply:
@rmckeel got it. Please change https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-widgets/index.d.ts#L1 this line to `4.4` (Please skip `patch` version, it will be set automatically). Yeah, I guess it would be better. :)

Reply:
@r3nya I bumped the version in my commit #31ebffe.  Looks like it added properly to this PR.

Reply:
Awesome! Thank you! :)